releasenotes/notes/transpiler-control-flow-708896bfdb51961d.yaml

```diff
@@ -0,0 +1,78 @@
+---
+features:
+  - |
+    Control-flow operations are now supported through the transpiler at
+    optimization levels 0 and 1.  One can now construct a circuit such as
+    
+    .. code-block:: python
+
+      from qiskit import QuantumCircuit
+
+      qc = QuantumCircuit(2, 1)
+      qc.h(0)
+      qc.measure(0, 0)
+      with qc.if_test((0, True)) as else_:
+        qc.x(1)
+      with else_:
+        qc.y(1)
+
+    and successfully transpile this, such as by::
+
+      from qiskit import transpile
+      from qiskit_aer import AerSimulator
+      
+      backend = AerSimulator(method="statevector")
+      transpiled = transpile(qc, backend)
+
+    The available layout methods are "trivial" and "dense".  For routing,
+    "stochastic" and "none" are available.  Translation can be done using
+    "translator" or "unroller".  Optimization levels 2 and 3 are not yet
+    supported with control flow, nor is circuit scheduling, though we intend to
+    expand support for these, and the other layout, routing and translation
+    methods in subsequent releases of Qiskit Terra.
+
+    In order for transpilation with control-flow operations to succeed with a
+    backend, the backend must have the requisite control-flow operations in its
+    stated basis.  Qiskit Aer, for example, does this.  If you simply want to try
+    out such transpilations, consider overriding the ``basis_gates`` argument
+    to :func:`.transpile`.
+  - |
+    The following transpiler passes have all been made control-flow aware:
```

Comment:
While in practice this will always immediately follow the above release note in the list I think it might be good to just add a sentence on what being control-flow aware means in this context.

Reply:
Done in a53a4a3fc.